code/datums/chatmessage.dm

```diff
@@ -184,14 +192,14 @@
   * * italics - Vacuum and other things
   * * size - Size of the message
   */
-/mob/proc/create_chat_message(atom/movable/speaker, raw_message, italics=FALSE, size)
+/mob/proc/create_chat_message(atom/movable/speaker, raw_message, italics=FALSE, size, symbol)
```

Comment:
Also since you're editing this already:
```suggestion
/mob/proc/create_chat_message(atom/movable/speaker, raw_message, italics = FALSE, size, symbol)
```